### PR TITLE
West Midlands Class 3 - Tenzin Samdup - Git and GitHub - Week 2

### PR DESCRIPTION
**Pull request by tenzyns**
Corrected answers for some questions.

# Your Details

- Your Name: Tenzin Samdup
- Your City: West Midlands
- Your Slack Name: Tenzyn

# Homework Details

- Module: Git & GitHub
- Week: 2

-----
[View rendered homework.md](https://github.com/tenzyns/GitHomeworkFixErrors/blob/main/homework.md)

### CODE REVIEW

**Comment by Professional-Kate** [comment]
I love how you created the ```<style>``` tags for the CSS question! Really wonderful answers, I'm satisfied! Really nice Job, Tenzin.

**Review by wiamnasr — APPROVED** [review]
Reviewed your pull request, thank you for your time

File: homework.md [hunk 1]
```diff
@@ -4,81 +4,83 @@
...
 
-Hyper Tall Massive Language
+Hyper Text Markup Language
 
 ## Question 5
 
 What is CSS useful for?
 
-Making a website go faster
+ Giving styles to HTML and make them display correctly.
```
Comment (gynaa78) [inline homework.md:32]: CSS is independent of HTML as it can be used with any XML- based markup language (w3C)

File: homework.md [hunk 2]
```diff
@@ -4,81 +4,83 @@
...
 
-You delete it
+You make a duplicate copy of the reposiory in your own account or device where changes made cannot affect the source file.
 
 ## Question 10 
 
 What happens when you clone a repostory?
 
-It send it to a friend
+It makes a duplicate copy on which changes made may affect the original file.
```
Comment (gynaa78) [inline homework.md:68]: It makes a full blown copy of a repo to your local computer.

**Review by gynaa78 — COMMENTED** [review]
Well done Tenzin!

**Pull request closed without merging**